rust/plugin-lib/src/lib.rs

```diff
@@ -107,9 +107,9 @@ pub trait Plugin {
 
     //TODO: async edits only; this is here for feature parity during initial hacking
     /// Called when an edit has occured in the remote view. If the plugin wishes
-    /// to add its own edit, it may return `Some(edit)`.
+    /// to add its own edit, it must do so using asynchronously via edit notification.
```

Comment:
Good stuff, it's important to keep docs up to date. I would say "via **the** edit notification".